docs/reference/components/sinks/amqp.cue

```diff
@@ -0,0 +1,106 @@
+package metadata
+
+components: sinks: amqp: {
+	title: "Amqp"
+
+	classes: {
+		commonly_used: true
+		delivery:      "at_least_once"
+		development:   "stable"
+		egress_method: "dynamic"
+		service_providers: ["AWS", "Confluent"]
+		stateful: false
+	}
+
+	features: {
+		buffer: enabled:      true
+		healthcheck: enabled: true
+		send: {
+			batch: {
+				enabled:      true
+				common:       true
+				max_bytes:    null
+				max_events:   null
+				timeout_secs: null
+			}
+			compression: {
+				enabled: true
+				default: "none"
+				algorithms: ["none", "gzip", "lz4", "snappy", "zstd"]
+				levels: ["none", "fast", "default", "best", 0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+			}
+			encoding: {
+				enabled: true
+				codec: {
+					enabled: true
+					default: null
+					enum: ["json", "text"]
+				}
+			}
+			request: enabled: false
+			tls: {
+				enabled:                true
```

Comment:
Does this source and sink TLS? I couldn't actually see where that was happening. It would be good to, if it isn't already.

Reply:
Thanks, had forgot to add that in.